### ISSUE TITLE
children.map is not a function

### ISSUE THREAD

**Issue by zero-** [STRML/react-grid-layout#21]
For some reason I can't find a way to pass children into the grid.

I'm trying to do this:
```
<Grid className="layout" cols={12}>
	<div key={1} _grid={{x: 1, y: 0, w: 1, h: 2, static: true}} >
		{this.props.children}
	</div>
</Grid>
```
but I keep getting ```ReactGridLayout.js:85  Uncaught TypeError: children.map is not a function``` error.

**Comment by STRML** [comment]
Thanks for the catch - `this.props.children` is not an array when only a single child is present, which is sort of a micro-optimization from the React team (one I don't really agree with). 

A fix is published in `0.6.2` which is available now.

**Comment by zero-** [comment]
Awesome fast fix, thanks :) Works perfectly now.
Didn't know about children not being an array with single child, not agreeing with the team either.
